Tests/Controller/AdminSecurityControllerTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\UserBundle\Tests\Controller;
+
+use Sonata\UserBundle\Controller\AdminSecurityController;
+use Sonata\UserBundle\Tests\Helpers\PHPUnit_Framework_TestCase;
+
+class AdminSecurityControllerTest extends PHPUnit_Framework_TestCase
```

Comment:
I added those with 2 purposes in mind:

- They do test object instantiation, so we can know if there is any syntax error or something like that.

- Having test created means that we can start testing step by step, asking contributors to test their changes.